### PR TITLE
Fixes #925 - Move Fluo documentation to project website

### LINKED ISSUE #925: Move Fluo documentation to project website

### PR DESCRIPTION
**Pull request by mikewalch**
Below is corresponding pull request to add docs to website repo.

https://github.com/apache/fluo-website/pull/83

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ of a large-scale computation, index, or analytic as new data is discovered. Chec
 ## Getting Started
 
 * Take the [Fluo Tour][tour] if you are completely new to Fluo.
-* Read the [install instructions][install] to install Fluo and start a Fluo application in YARN on a
-  cluster where Accumulo, Hadoop & Zookeeper are running. If you need help setting up these
+* Read the [Fluo documentation][fluo-docs] to learn how to install Fluo and start a Fluo application
+  on a cluster where Accumulo, Hadoop & Zookeeper are running. If you need help setting up these
   dependencies, see the [related projects page][related] for external projects that may help.
 
 ## Applications
```
Comment (keith-turner) [inline README.md]: Thinking maybe should remove this section?

File: README.md [hunk 2]
```diff
@@ -42,25 +42,15 @@ Below are helpful resources for Fluo application developers:
...
-   or Ganglia
-
 [fluo]: https://fluo.apache.org/
 [related]: https://fluo.apache.org/related-projects/
 [tour]: https://fluo.apache.org/tour/
 [accumulo]: https://accumulo.apache.org
 [percolator]: https://research.google.com/pubs/pub36726.html
-[install]: docs/install.md
-[apps]: docs/applications.md
+[fluo-docs]: https://fluo.apache.org/docs/fluo/1.1.0-incubating/
```
Comment (keith-turner) [inline README.md]: What about linking to `https://fluo.apache.org/docs/` instead of a specific version?

File: README.md (hunk 1 above)
Comment (mikewalch) [reply README.md]: I agree.  We should just point users to documentation on the website.

File: README.md (hunk 2 above)
Comment (mikewalch) [reply README.md]: Good idea.

**Pull request merged**